docs/avatar.md

```diff
@@ -273,23 +250,22 @@ Makes the avatar circular
 
 ---
 
-### `small`
+### `source`
 
-Small sized avatar
+Image source
 
-|  Type   | Default |
-| :-----: | :-----: |
-| boolean |  false  |
+|      Type      | Default |
+| :------------: | :-----: |
+| object (style) |  none   |
 
 ---
+### `size`
 
-### `source`
-
-Image source
+Size of the avatar
 
 |      Type      | Default |
 | :------------: | :-----: |
-| object (style) |  none   |
+| string(`small`, `medium`, `large`, `xlarge`) |  `small`  |
```

Comment:
type would be string(`small`, `medium`, `large`, `xlarge`) OR number

Reply:
yeah I changed that for the next pul request